Orange/widgets/unsupervised/owkmeans.py

```diff
@@ -466,6 +471,16 @@ def invalidate(self, unconditional=False):
         else:
             self.commit.deferred()
 
+    def on_init_changed(self):
+        self.invalidate()
+
+        self.reruns.setEnabled(True)
+        if isinstance(self.data, DaskTable):
+            if self.INIT_METHODS[self.smart_init][1] != "k-means||":
+                self.Warning.default_to_sklearn(self.INIT_METHODS[self.smart_init][1])
```

Comment:
A warning is good, but I'd rather have the option to come here at all disabled. Initialization is probably not that big of a deal that we'd want a specific method so badly.